test/cts/test/V1_3/conv2d_quant8_signed.js

```diff
@@ -0,0 +1,510 @@
+    let type88_length = product(type88.dimensions);
+    let type89 = {type: nn.TENSOR_QUANT8_SYMM_PER_CHANNEL, dimensions: [3, 1, 1, 3]};
+    let type89_length = product(type89.dimensions);
+    let type90 = {type: nn.TENSOR_INT32, dimensions: [3], scale: 0.0, zeroPoint: 0};
+    let type90_length = product(type90.dimensions);
+
+    let op110 = operandIndex++;
+    model.addOperand(type86);
+    let op210 = operandIndex++;
+    model.addOperand(type89);
+    model.setOperandSymmPerChannelQuantParams(op210, {channelDim: 0, scales: new Float32Array([0.5, 1.0, 0.5])});
+    let op310 = operandIndex++;
+    model.addOperand(type90);
+    let param88 = operandIndex++;
+    model.addOperand(type4);
+    let param89 = operandIndex++;
+    model.addOperand(type4);
+    let param90 = operandIndex++;
+    model.addOperand(type4);
+    let param91 = operandIndex++;
+    model.addOperand(type4);
+    let param92 = operandIndex++;
+    model.addOperand(type4);
+    let param93 = operandIndex++;
+    model.addOperand(type4);
+    let param94 = operandIndex++;
+    model.addOperand(type4);
+    let op410 = operandIndex++;
+    model.addOperand(type88);
+
+    model.setOperandValue(op210, new Int8Array([2, 8, 14, 2, 5, 8, 6, 12, 18]));
+    model.setOperandValue(op310, new Int32Array([0, 0, 0]));
+    model.setOperandValue(param88, new Int32Array([0]));
+    model.setOperandValue(param89, new Int32Array([0]));
+    model.setOperandValue(param90, new Int32Array([0]));
+    model.setOperandValue(param91, new Int32Array([0]));
+    model.setOperandValue(param92, new Int32Array([1]));
+    model.setOperandValue(param93, new Int32Array([1]));
+    model.setOperandValue(param94, new Int32Array([0]));
+    model.addOperation(nn.CONV_2D, [op110, op210, op310, param88, param89, param90, param91, param92, param93, param94], [op410]);
+
+    model.identifyInputsAndOutputs([op110], [op410]);
+    await model.finish();
+
+    let compilation = await model.createCompilation();
+    compilation.setPreference(getPreferenceCode(options.prefer));
+    await compilation.finish();
+
+    let execution = await compilation.createExecution();
+
+    let op110_input = new Int8Array(op110_value);
+    execution.setInput(0, op110_input);
+    let op410_output = new Int8Array(type88_length);
+    execution.setOutput(0, op410_output);
+
+    await execution.startCompute();
+
+    for (let i = 0; i < type88_length; ++i) {
+      assert.isTrue(almostEqualCTS(op410_output[i], op410_expect[i]));
+    }
+  });
+
+  it('check result for Conv2d quant8 signed example-7', async function() {
+    // For 'Conv2d quant8 signed' example: examples_large_nhwc_channelQuant8_2
+    let model = await nn.createModel(options);
+    let operandIndex = 0;
+
+    let op110_value = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17];
+    let op410_expect = [29, 35, 41, 65, 80, 95, 101, 125, 149, 137, 170, 203, 173, 215, 257, 209, 260, 311];
```

Comment:
The type of output operand is `TENSOR_QUANT8_ASYMM_SIGNED`, its value should be in [-128, 127]. Could you please clarify why the expect output has values greater than 127? Thanks

Reply:
Yeah, you are right. I miss to quantize '`TENSOR_QUANT8_ASYMM_SIGNED`' type data.
Base: https://android.googlesource.com/platform/frameworks/ml/+/refs/heads/master/nn/tools/test_generator/test_generator.py#78